README.md

```diff
@@ -1,18 +1,32 @@
+[![This project uses GitHub Actions for continuous integration.](https://github.com/ember-learn/guidemaker-ember-template/workflows/CI/badge.svg)](https://github.com/ember-learn/guidemaker-ember-template/actions?query=workflow%3ACI)
+[![This project uses Percy.io for visual regression testing.](https://percy.io/static/images/percy-badge.svg)](https://percy.io/Ember/guidemaker-ember-template)
+
 # guidemaker-ember-template
 
-[Short description of the addon.]
+This addon provides the general layout and some styles that are used for documentation websites such as [guides.emberjs.com](https://guides.emberjs.com/release/), and [cli.emberjs.com](https://cli.emberjs.com/release/).
 
-## Compatibility
+Its responsibilities include
+- CSS for the sidebar
+- HTML markup specifics (i.e. the classes that go on various divs in the text of the guide itself)
 
-- Ember.js v4.12 or above
-- Embroider or ember-auto-import v2
+## Related apps
+
+Most aspects of the Guides are styled via [ember-styleguide](https://github.com/ember-learn/ember-styleguide), not this repository.
+
+[guidemaker](https://github.com/empress/guidemaker) is what assembles guides content into queryable records.
+
+[guides-source](https://github.com/ember-learn/guides-source) has the learning content itself.
 
 ## Installation
 
 ```
 ember install guidemaker-ember-template
 ```
 
+## Compatibility
+- Ember.js v4.12 or above
```

Comment:
```suggestion
- Ember.js v3.28 or above
```

This should always reflect what is tested in the Github Action, and I always leave more than the default blueprint gives you 👍

Reply:
Thanks for the comment. Makes sense, i'll try to remember for next times